### ISSUE TITLE
Konami japanese games won't run on below Android 5 devices

### ISSUE THREAD

**Issue by Yabuato** [libretro/nestopia#82]
I use the same set up,tested on 3 devices
Result: 

Device 1 - Android 4.2.2 - failed black screen i.imgur.com/t4fx477.jpg

Device 2 - Android 4.4.2 - failed black screen https://i.imgur.com/ojAODMA.jpg

Device 3 - Android 5.0 - worked https://i.imgur.com/7ghkoev.jpg

**Comment by meepingsnesroms** [comment]
You also state that the android 5 device is 64bit, have you tried a 32bit android >= 5  device?

64bit is far more likely the issue given that non fixed with int types are used in alot of the mapper code.

typedef signed char schar;
typedef unsigned char uchar;
typedef unsigned short ushort;
typedef unsigned int uint;
typedef unsigned long ulong;

**Comment by Yabuato** [comment]
@meepingsnesroms  Thanks for reply first of all.I can't test it on a 32bit android >= 5 device since I don't have one.

**Comment by meepingsnesroms** [comment]
That file says no longer available?

**Comment by Yabuato** [comment]
@meepingsnesroms Oops,I thought you have already downloaded the file.
.
New link: https://mega.nz/#!fNRknRgR!JEy6m9bQ1Ldh3csE4tqSCUKwDlnOCii_W5wSTq-2rWM

**Comment by meepingsnesroms** [comment]
That log shows nothing of inters but the bug I described would not show up on a log, I will look into this in the next week.

**Comment by Yabuato** [comment]
Sure,MD5 checksum of my Contra (Japan).nes

0E40BC1B049C16C5D7246CC28399CB5D

**Comment by meepingsnesroms** [comment]
I used a Moto E running Android 7.1.2 and it did failed to load content and if I loaded it again crashed, this also happens on Mac OS.

Your newer phones must be using a older version of nestopia since with the latest nightly it happens on all devices, it works properly on FCEUmm though.

**Comment by Yabuato** [comment]
Yes,I used same setup for my test and I have NstDatabase.xml in my system/bios folder on my three devices,they use the same RA version,same nestopia core from same commit.
